<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibration</class>
 <widget class="QWidget" name="Calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>200</y>
     <width>384</width>
     <height>384</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border-image: url(:/Reconstruction/Resources/checkered.png)}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1221</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QLabel" name="label4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label11">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label12">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>650</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>读取标定图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>650</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>标定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Reconstruction.qrc"/>
 </resources>
 <connections/>
</ui>
